lib/cisco_node_utils/bgp.rb

```diff
@@ -691,16 +653,15 @@ def default_reconnect_interval
       config_get_default('bgp', 'reconnect_interval')
     end
 
-    # Route Distinguisher (Getter/Setter/Default)
-    # Configure in vrf context
+    # route_distinguisher
+    # Note that this property is supported by both bgp and vrf providers.
     def route_distinguisher
-      return false unless RouterBgpAF.feature_nv_overlay_evpn_enabled
       config_get('bgp', 'route_distinguisher', @get_args)
     end
 
     def route_distinguisher=(rd)
-      RouterBgpAF.feature_nv_overlay_evpn_enable unless
-        RouterBgpAF.feature_nv_overlay_evpn_enabled
+      Feature.nv_overlay_enable
+      Feature.nv_overlay_evpn_enable
```

Comment:
Yeah, considered that. Also thought about maybe getting rid of nv_overlay_evpn_enable altogether and just making it part of nv_overlay_enable since it's not really a "feature" command (`nv overlay evpn`). I think I'd prefer to keep it like this until we can come up with a better story for it.